Makefile

```diff
@@ -1,13 +1,17 @@
 NAME = $(shell awk -F\" '/^const Name/ { print $$2 }' cmd/root.go)
-VERSION = $(shell awk -F\" '/^const Version/ { print $$2 }' cmd/version.go)
 RPCLINT=$(shell find ./rpc -type f \( -not -iname 'root.*.go' -iname '*.go' \) )
 TOLINT = $(shell find . -type f \( -not -ipath './vendor*'  -not -ipath './docs*' -not -ipath './rpc*' -not -iname 'main.go' -iname '*.go' \) -exec dirname {} \; | sort -u)
 TESTDIRS = $(shell find . -name '*_test.go' -exec dirname \{\} \; | grep -v vendor | uniq)
 NONVENDOR = ${shell find . -name '*.go' | grep -v vendor}
 BENCHDIRS= $(shell find . -name '*_test.go' | grep -v vendor | xargs grep '*testing.B' | cut -d: -f1 | xargs -n1 dirname | uniq)
 BENCH = .
 
-converge: $(shell find . -name '*.go') rpc/pb/root.pb.go rpc/pb/root.pb.gw.go
+converge: $(shell find . -name '*.go' -not -name 'version_number.go') rpc/pb/root.pb.go rpc/pb/root.pb.gw.go
+	@echo "package cmd" > cmd/version_number.go
```

Comment:
We could make this a phony target and depend on it for the converge target just to keep version number generation separate from building, but it's probably not that big of a deal.